conda-forge.yml

```diff
@@ -1,5 +1,5 @@
 bot:
-  inspection: hint-all
+  inspection: update-grayskull
```

Comment:
I hope everyone's okay with this change.  I think it's going to be a lot more useful.

Reply:
Hmm, I see. Actually `hint-all` might be better.  I'll change it back.